app/connectors_service/connectors/sources/mongo/mongoconnectorutils.py

```diff
@@ -0,0 +1,76 @@
+#
```

Comment:
Should probably be just renamed to `utils`?

Also I see you normally move validators to separate files, should probably happen here too